### PR TITLE
Fix prefix static source 404 issue

### PR DESCRIPTION
**Pull request by acured**

### CODE REVIEW

**Review by Lijiaoa — APPROVED** [review]

File: ts/nni_manager/rest_server/nniRestServer.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class NNIRestServer extends RestServer {
      * NNIRestServer's own router registration
      */
     protected registerRestHandler(): void {
-        this.app.use(express.static('static'));
+        this.app.use(getPrefixUrl(), express.static('static'))
```
Comment (liuzhe-lz) [inline ts/nni_manager/rest_server/nniRestServer.ts:41]: Missing a semicolon here.

File: ts/nni_manager/common/experimentStartupInfo.ts [hunk 2]
```diff
@@ -126,3 +126,8 @@ export function getDispatcherPipe(): string | null {
 export function getAPIRootUrl(): string {
     return getExperimentStartupInfo().apiRootUrl;
 }
+
+export function getPrefixUrl(): string {
+    const prefix = getExperimentStartupInfo().urlprefix === ''?'':`/${getExperimentStartupInfo().urlprefix}`;
```
Comment (liuzhe-lz) [inline ts/nni_manager/common/experimentStartupInfo.ts:131]: Please add spaces around operators to make the style in sync with other codes.

Comment (Lijiaoa) [reply ts/nni_manager/common/experimentStartupInfo.ts:131]: why not add lint for this format in next release?

File: ts/nni_manager/rest_server/nniRestServer.ts (hunk 1 above)
Comment (acured) [reply ts/nni_manager/rest_server/nniRestServer.ts]: fixed.

File: ts/nni_manager/common/experimentStartupInfo.ts (hunk 2 above)
Comment (acured) [reply ts/nni_manager/common/experimentStartupInfo.ts]: fixed.

**Review by liuzhe-lz — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]

**Pull request merged**